lib/Braintree/Configuration.php

```diff
@@ -254,7 +254,7 @@ public function merchantPath()
      */
     public function caFile($sslPath = NULL)
     {
-        $sslPath = $sslPath ? $sslPath : DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR .
+        $sslPath = $sslPath ? $sslPath : DIRECTORY_SEPARATOR .
                    'ssl' . DIRECTORY_SEPARATOR;
```

Comment:
So we don't install the SSL CA files in a non-name-spaced directory in the PEAR lib dir. If we leave this as-is and another package does the same thing, they could overwrite each other.